package.json

```diff
@@ -60,19 +60,33 @@
         {
           "id": "spring.beans",
           "name": "Beans",
-          "when": "java:serverMode",
+          "when": "java:serverMode && !spring:actuatorRequired",
           "contextualTitle": "Spring",
           "icon": "resources/logo.png"
         },
         {
           "id": "spring.mappings",
           "name": "Endpoint Mappings",
-          "when": "java:serverMode",
+          "when": "java:serverMode && !spring:actuatorRequired",
```

Comment:
What's the purpose of checking `java:serverMode`?

Reply:
hide for non-java workspaces